cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -599,7 +602,8 @@ String pmUrl(String privacyManagerId, String pmTab) {
         params.add("pmTab="+pmTab);
         params.add("site_id="+ propertyId);
         if (consentUUID != null) params.add("consentUUID=" + consentUUID);
-        return PM_BASE_URL + "?" + TextUtils.join("&", params);
+        String PM_URL = isOTT ? OTT_PM_BASE_URL : PM_BASE_URL;
+        return PM_URL + "?" + TextUtils.join("&", params);
```

Comment:
It'd be cool to test that if the `isOTT` flag is set, the WebView gets loaded with the correct URL.

Reply:
@andresilveirah I have added unit test in ConsentLibBuilder class to check isOTT flag set

Reply:
@Suryakantj right, you added a spec for the setter. I'm referring to the actual logic of changing from one PM url to another based on that flag.

Reply:
@andresilveirah As suggested I have added unit to check correct pmURL.